app.rb

```diff
@@ -0,0 +1,63 @@
+require_relative './teacher'
+require_relative './student'
+require_relative './rental'
+require_relative './book'
+require_relative './person'
+
+class App
+  def initialize
+    @books = []
+    @peoples = []
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: #{book.title} Author: #{book.author} \n" }
+  end
+
+  def list_peoples
+    @peoples.each do |people|
+      puts "[#{people.class.name}] Name: #{people.name} ID: #{people.id} Age: #{people.age} \n"
+    end
+  end
+
+  def create_teacher(age, specialization, parent_permission, name)
+    @peoples << Teacher.new(age, specialization, parent_permission: parent_permission, name: name)
+  end
+
+  def create_student(age, name, parent_permission)
+    @peoples << Student.new(age, name, parent_permission: parent_permission)
+  end
```

Comment:
- [ ] @MuhammadUsmanAslam great work on your app, however, am unable to create a new person either a teacher or student am getting the errors below 
![image](https://github.com/MuhammadUsmanAslam/OOP-school-library/assets/61408860/67d44b75-a27b-4e81-bb5d-396d4590ea5a)
![image](https://github.com/MuhammadUsmanAslam/OOP-school-library/assets/61408860/9cdd908b-380f-4066-8393-6fd8811dfc25)